### PR TITLE
Large video zooming

### PR DESCRIPTION
**Pull request by hristoterezov**

### CODE REVIEW

File: modules/UI/videolayout/VideoContainer.js [hunk 1]
```diff
@@ -78,44 +78,44 @@ function computeDesktopVideoSize(videoWidth,
...
-        availableHeight = availableWidth / aspectRatio;
-    } else if (interfaceConfig.VIDEO_LAYOUT_FIT === 'both') {
-        availableWidth = Math.max(videoWidth, videoSpaceWidth);
-        availableHeight = Math.max(videoHeight, videoSpaceHeight);
-
-        if (availableWidth / aspectRatio < videoSpaceHeight) {
-            availableHeight = videoSpaceHeight;
-            availableWidth = availableHeight * aspectRatio;
+    switch (videoLayoutFit) {
+    case 'height':
```
Comment (virtuacoplenny) [inline modules/UI/videolayout/VideoContainer.js]: Nice cleanup.

File: css/_videolayout_default.scss [hunk 2]
```diff
@@ -12,10 +12,7 @@
     overflow: hidden;
 }
 
-.video_blurred {
-    width: 100%;
-
-    &_container {
+.video_blurred_container {
```
Comment (virtuacoplenny) [inline css/_videolayout_default.scss]: Looks like there's a dangling curly brace left in this block.

File: interface_config.js [hunk 3]
```diff
@@ -93,5 +93,6 @@ var interfaceConfig = { // eslint-disable-line no-unused-vars
      *
      * @type {boolean}
      */
-    MOBILE_APP_PROMO: true
+    MOBILE_APP_PROMO: true,
+    MAXIMUM_ZOOMING_COEFFICIENT: 1.3
```
Comment (saghul) [inline interface_config.js]: can you add some docs here?

Comment (hristoterezov) [inline interface_config.js]: Done!

File: css/_videolayout_default.scss (hunk 2 above)
Comment (hristoterezov) [inline css/_videolayout_default.scss]: Done!

**Comment by virtuacoplenny** [comment]
Changes look good to me but I did notice some potential UX/UI issues while testing that aren't related to the latest changes. Maybe @yanas needs to voice her opinion on them.
- With horizontal filmstrip, the blur effect is still visible under YouTube sharing and etherpad.
- With horizontal filmstrip, screenshares do not stretch to the bottom of the screen so maybe it can look off to someone when coupled with the blur effect.

**Comment by yanas** [comment]
-> With horizontal filmstrip, the blur effect is still visible under YouTube sharing and etherpad.
@hristoterezov isn't this easily fixable by checking if we have the VIDEO_CONTAINER_TYPE on large?
-> With horizontal filmstrip, screenshares do not stretch to the bottom of the screen so maybe it can look off to someone when coupled with the blur effect.
@virtuacoplenny did you mean with vertical filmstrip? Do you have a screenshot for this one? I'm not sure I completely see what the problem is.

**Comment by hristoterezov** [comment]
> > With horizontal filmstrip, the blur effect is still visible under YouTube sharing and etherpad.
@hristoterezov isn't this easily fixable by checking if we have the VIDEO_CONTAINER_TYPE on large?

@yanas Yeah, it should be easy. I'll give it a try now! Just wanted to make sure this is what we want!

**Comment by virtuacoplenny** [comment]
I was referring to this appearance but on second thought it looks okay to me.
screen shot 2017-06-15 at 11 12 01 am

**Comment by hristoterezov** [comment]
@yanas @virtuacoplenny The issue with the shared video and background video is fixed! If you don't have any other comments we should be ready for merge!

File: modules/UI/videolayout/VideoContainer.js [hunk 4]
```diff
@@ -214,6 +214,13 @@ export class VideoContainer extends LargeContainer {
 
         this.$wrapper = $('#largeVideoWrapper');
 
+        // FIXME: the architecture of elements related to the large video and
+        // the naming. The background is not part of largeVideoWrapper because
+        // we are controlling the size of the video trough largeVideoWrapper.
```
Comment (virtuacoplenny) [inline modules/UI/videolayout/VideoContainer.js]: I don't completely understand this comment. What is the relationship between controlling the size of the video and the background?

**Pull request merged**